Dockerfile

```diff
@@ -1,8 +1,7 @@
-FROM fedora:22
-MAINTAINER mscherer@redhat.com
+FROM fedora:23
+MAINTAINER jberkus@redhat.com
```

Comment:
I am not sure we have generic projectatomic address. The domain is handled by IT, so we would have to see with them to add the alias. But then, that's more question of who have the ownership of the file in the sense "who is tasked to keep it updated".